webapp/src/shared/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,53 @@
+import './Avatar.css';
+
+import { Icon, IconVariant, IconSize } from '../index';
+import { Color } from '../../../shared/types';
+
+type AvatarStatus = 'online' | 'offline' | 'playing';
```

Comment:
This type is temporal until we have `status` component integrated. The idea is to be able to import `StatusType` here (and everywhere using statuses) so we have a centralized defintion of it

Reply:
done!